diff --git a/Makefile b/Makefile
index d614bfa..feddd8c 100644
--- a/Makefile
+++ b/Makefile
@@ -8,8 +8,8 @@ ifeq ($(INTEL), 1)
     CXX ?= /opt/intel/bin/icpc
     CC ?= /opt/intel/bin/icpc
 ifeq ($(DEBUG),1)
-    CXXFLAGS = -fpic -std=c++11 -O3 -Wall -ansi -xAVX -DDEBUG=1 -D_GLIBCXX_DEBUG   -ggdb
-    CCFLAGS = -fpic -std=c99 -O3 -Wall  -ansi -xAVX -DDEBUG=1 -D_GLIBCXX_DEBUG   -ggdb
+    CXXFLAGS = -fpic -std=c++11 -O3 -Wall -ansi -xAVX -DDEBUG=1 -ggdb
+    CCFLAGS = -fpic -std=c99 -O3 -Wall  -ansi -xAVX -DDEBUG=1 -ggdb
 else
     CXXFLAGS = -fpic -std=c++11 -O2 -Wall  -ansi -xAVX -DNDEBUG=1  -ggdb
     CCFLAGS = -fpic -std=c99 -O2 -Wall   -ansi -xAVX -DNDEBUG=1  -ggdb
@@ -17,8 +17,8 @@ endif # debug
 else #intel
     CXX ?= g++-4.7
 ifeq ($(DEBUG),1)
-    CXXFLAGS = -fpic -mavx -std=c++11  -Weffc++ -pedantic -ggdb -DDEBUG=1 -D_GLIBCXX_DEBUG -Wall  -Wextra -Wextra -Wsign-compare  -Wwrite-strings -Wpointer-arith -Winit-self  -Wno-sign-conversion
-    CCFLAGS = -fpic -mavx -std=c99  -pedantic -ggdb -DDEBUG=1 -D_GLIBCXX_DEBUG -Wall  -Wextra -Wsign-compare -Wwrite-strings -Wpointer-arith -Winit-self  -Wno-sign-conversion
+    CXXFLAGS = -fpic -mavx -std=c++11  -Weffc++ -pedantic -ggdb -DDEBUG=1 -Wall  -Wextra -Wextra -Wsign-compare  -Wwrite-strings -Wpointer-arith -Winit-self  -Wno-sign-conversion
+    CCFLAGS = -fpic -mavx -std=c99  -pedantic -ggdb -DDEBUG=1 -Wall  -Wextra -Wsign-compare -Wwrite-strings -Wpointer-arith -Winit-self  -Wno-sign-conversion
 else
     CXXFLAGS = -fpic -mavx -std=c++11  -Weffc++ -pedantic -O3 -Wall  -Wextra -Wsign-compare  -Wwrite-strings -Wpointer-arith -Winit-self  -Wno-sign-conversion
     CCFLAGS = -fpic -mavx -std=c99 -pedantic -O3 -Wall -Wextra -Wsign-compare -Wwrite-strings -Wpointer-arith -Winit-self -Wno-sign-conversion
diff --git a/src/frameofreference.cpp b/src/frameofreference.cpp
index af76e82..43d187f 100644
--- a/src/frameofreference.cpp
+++ b/src/frameofreference.cpp
@@ -1,6 +1,6 @@
 #include "frameofreference.h"
 
-static __m128i *simdpackFOR_length(uint32_t initvalue, const uint32_t *in,
+__m128i *simdpackFOR_length(uint32_t initvalue, const uint32_t *in,
                                    int length, __m128i *out,
                                    const uint32_t bit) {
   int k;
@@ -56,7 +56,7 @@ static __m128i *simdpackFOR_length(uint32_t initvalue, const uint32_t *in,
   return out;
 }
 
-static const __m128i *simdunpackFOR_length(uint32_t initvalue,
+const __m128i *simdunpackFOR_length(uint32_t initvalue,
                                            const __m128i *in, int length,
                                            uint32_t *out, const uint32_t bit) {
   int k;
@@ -27907,7 +27907,7 @@ static uint32_t maxasint(const __m128i accumulator) {
   return _mm_cvtsi128_si32(_tmp2);
 }
 
-static void simdmaxmin_length(const uint32_t *in, uint32_t length,
+void simdmaxmin_length(const uint32_t *in, uint32_t length,
                               uint32_t *getmin, uint32_t *getmax) {
   uint32_t lengthdividedby4 = length / 4;
   uint32_t offset = lengthdividedby4 * 4;
